support/dynamics/gp/year-end-closing-payables-management.md

```diff
@@ -174,129 +172,120 @@ To create a backup in Microsoft Dynamics GP, follow these steps:
 2. In the **Company Name** list, select your company name.
 3. Change the path of the backup file if it's required, and then select **OK**.
 
-    > [!NOTE]
-    > We recommend that you name this backup **Pre-1099 Edits** to differentiate it from your other backups.
+   > [!NOTE]
+   > We recommend that you name this backup "Pre-1099 Edits" to differentiate it from your other backups.
 
 ### Step 6: Verify the 1099 information and edit it if it's required
 
 To print the 1099 edit list, follow these steps:
 
-1. In Microsoft Dynamics GP, point to **Tools** on the **Microsoft Dynamics GP** menu, point to **Routines**, point to **Purchasing**, and then select **Print 1099**.
+1. In Microsoft Dynamics GP, point to **Tools** on the **Microsoft Dynamics GP** menu, point to **Routines** > **Purchasing**, and then select **Print 1099**.
 
 2. Enter the Company Information and Address ID.
 
-3. Select the printer icon in the upper right corner to print the 1099 Edit List. Review for accuracy.
+3. Select the printer icon in the upper-right corner to print the 1099 edit list. Review for accuracy.
 
-4. If you need to edit any 1099 information, use one of the following methods (Method 1 and 2 is recommended for Microsoft Dynamics GP 2013 and higher versions and also illustrated in [Editing made EASY for 1099s in Microsoft Dynamics GP](https://community.dynamics.com/blogs/post/?postid=487c4288-7473-47de-a4e7-221c26a2adc6)
+4. If you need to edit any 1099 information, use one of the following methods. Method 1 is recommended for Microsoft Dynamics GP 2013 and higher versions. Method 1 and Method 2 are also illustrated in [Editing made EASY for 1099s in Microsoft Dynamics GP](https://community.dynamics.com/blogs/post/?postid=487c4288-7473-47de-a4e7-221c26a2adc6).
```

Comment:
Hi @theley502, I made some changes to line 188 based on your changes to method 2 (see line 207). Could you have a check and let me know if my understanding is not correct? Thanks!